wagtail/admin/templates/wagtailadmin/pages/bulk_actions/confirm_bulk_delete.html

```diff
@@ -50,7 +50,17 @@
     {% if items and not is_protected %}
         {% trans 'Yes, delete' as action_button_text %}
         {% trans "No, don't delete" as no_action_button_text %}
-        {% include 'wagtailadmin/bulk_actions/confirmation/form.html' with action_button_class="serious" %}
+        {% if type_to_confirm_before_delete %}
+            <form method="POST">
```

Comment:
I'd suggest an adjustment here so that we can reuse the `<form>` and button elements from `wagtailadmin/bulk_actions/confirmation/form.html`, so that we don't risk bugs creeping in through subtle differences in the two forms (such as the missing `action="{{ submit_url }}"` above):

Create a new template `wagtailadmin/pages/bulk_actions/confirm_delete_form.html` consisting of

```
{% extends 'wagtailadmin/bulk_actions/confirmation/form.html' %}
{% block form_fields %}
    {% if type_to_confirm_before_delete %}
        {% include "wagtailadmin/shared/dialog/confirm_site_name.html" with total_pages=total_page_count %}
    {% endif %}
{% endblock form_fields %}
```

and then include this template in lieu of `wagtailadmin/bulk_actions/confirmation/form.html`.